src/python/T0/RunConfig/Tier0Config.py

```diff
@@ -573,6 +573,18 @@ def setProcessingSite(config, site):
     config.Global.ProcessingSite = site
     return
 
+def setStreamMem(config, streamname, **options):
+    """
+    _SetStreamMem_
+    Define memory usage for any type of streamer configurable.
+    The following keys may be passed:
+        value - size of memory to be assigned 
+    """
+    if 'value' in options:
+        config.Global.StreamMem = options['value']
```

Comment:
You are changing `StreamMem` for the global configuration. This means you are not really allowing us to configure different memory values for different streamers, just changing the default value.